recipes/flit-core/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "3.9.0" %}
+
+package:
+  name: flit-core
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/f/flit/flit-{{ version }}.tar.gz
+  sha256: d75edf5eb324da20d53570a6a6f87f51e606eee8384925cd66a90611140844c7
+  folder: source
+
+build:
+  script: python -m pip install --no-deps -vv source/flit_core/
+  noarch: python
+  number: 1
+
+requirements:
+  host:
+    - python >=3.6
```

Comment:
I'd kinda prefer making this `python >=3.8`; we're not testing anything on python versions below that anymore, so IMO we shouldn't be pushing packages to people who are still on <3.8.